src/extension.ts

```diff
@@ -150,11 +146,25 @@ function initIntelliSense(context: vscode.ExtensionContext): void {
   );
   context.subscriptions.push(
     vscode.workspace.onDidCloseTextDocument(document => {
-      if (isJsonFile(document)) {
+      if (IntelliSenseUtility.isJsonFile(document)) {
         diagnosticCollection.delete(document.uri);
       }
     })
   );
+  // send DigitalTwin usage telemetry
+  context.subscriptions.push(
+    vscode.workspace.onDidOpenTextDocument(document => {
+      if (IntelliSenseUtility.isJsonFile(document)) {
+        const modelContent: ModelContent | undefined = IntelliSenseUtility.parseDigitalTwinModel(document.getText());
+        if (modelContent) {
+          const telemetryContext: TelemetryContext = telemetryWorker.createContext();
+          telemetryContext.properties.result = TelemetryResult.Succeeded;
+          telemetryContext.properties.version = modelContent.version.toString();
```

Comment:
can `version` be something like `DTDLVersion`? This could help people quickly understand what does this property mean with its name.

Reply:
fixed. Change to dtdlVerson